### DISCUSSION TITLE
Final Assessment (Week 13)

### DISCUSSION THREAD

**Discussion by coopsand** [USU-ITLS-x265-23-summer/class-repo#247] [General]
Hello everyone!

- [Final Site](https://usu-itls-x265-23-summer.github.io/coopsandportfolio/)
- [Repo](https://github.com/USU-ITLS-x265-23-summer/coopsandportfolio/tree/main)

Here is a breakdown of my project. I created a portfolio website to showcase my work. You will find my exact focus areas below:

**HTML (Beginner):**
The HTML I wrote for this site was not very complex. I did have to remind myself how to do a couple of things, and having never used HTML prior to this class, I am proud of what I accomplished.

**CSS (Beginner):**
I used one CSS sheet for my entire project. This was what made the most sense to me, especially for a portfolio website like this one. One of my favorite things I did was adding in CSS for when the mouse hovers over an item in the navigation bar. I chose to make it so that the cursor would appear as a pointer finger, and the color of text would change. This made it clear when the user hovered over an item in the list.

**GIT (Intermediate):**
Rather than using the built in tools to `commit` and `push`, I used the terminal. I wanted to learn a little more about how you would `commit` and `push` in a more traditional aspect. I shared some of the things I learned [here](https://github.com/USU-ITLS-x265-23-summer/class-repo/blob/main/References/Project-/Project-Help.md). (The sections I added are titled "I can't see my edits inside of GitHub?! Help!!" and "How do I know if my project is up-to-date between Visual Studio and GitHub?")

**Accessibility (Intermediate):**
Every image on my site has alternate text, and the colors I used have a high contrast to comply with WCAG AA standards. (See screenshots below.)
<img width="395" alt="Screenshot 2023-08-07 at 2 12 54 PM" src="https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133150498/32e35a76-c397-420d-bfb6-216467a84ede">
<img width="395" alt="Screenshot 2023-08-07 at 2 13 43 PM" src="https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133150498/5ff27e5d-f399-436d-843f-eec788349ece">

**UX Design (Intermediate):**
I split my portfolio into separate pages, to make it easier to navigate. The navigation bar is on all pages, and links to every page. I tried to organize the pieces in a way that made sense, and used headings to split the categories up, so the user would know what bio goes with what image.

**In Conclusion:**
This class really helped me gain confidence in HTML, and I hope to continue learning in the future.

**Comment by gabebott** [comment]
Hey Cooper great job on your website. I also really enjoyed your breakdown of the topics. I think that your website flowed very smoothly, as well as being very visually appealing. This is a really cool and easy to navigate portfolio. Great job!

**Comment by kassyoveson** [comment]
Your final project turned out great. I saw just a few little changes from last week. Enjoy the rest of your summer.
